tests/ApiCallTest.php

```diff
@@ -869,6 +869,171 @@ public function testGlobalMockException()
         MockHelper::responseHandler()->getResult($context);
     }
 
+    public function testJsonPointersWithJsonArrayTypePointer()
+    {
+        $this->expectExceptionMessage(
```

Comment:
Also Assert exception type using `$this->expectException(MockException1::class);` before asserting exception messages in all the newly add unit tests